util/guard-one/guard-one.repl

```diff
@@ -0,0 +1,172 @@
+;; gas-guards.repl
+(env-data
+ { 'util-ns-users: ["util-ns-user"]
+ , 'util-ns-admin: ["util-ns-admin"]
+ })
+
+(env-keys ["util-ns-user", "util-ns-admin"])
+
+;;Load contracts
+(begin-tx)
+(load "../util/util-ns.pact")
+(load "../guards/guards.pact")
+(load "guard-one.pact")
+(commit-tx)
+
+(begin-tx)
+(use util.guard-one)
+
+; set chain gas price to 0.0005
+(env-chain-data {
+  "gas-price": 0.0005,
+  "gas-limit": 500
+})
+
+;;Test max-gas
+
+(expect
+  (format "chain-gas {} is smaller than max-gas {}"
+    [(chain-gas), 0.5]) true
+  (enforce-guard
+    (max-gas 0.5)))
+
+(expect
+  (format "chain-gas {} is at than max-gas {}"
+    [(chain-gas), 0.25]) true
+  (enforce-guard
+    (max-gas 0.25)))
+
+(expect-failure
+  (format "chain-gas {} is bigger than than max-gas {}"
+    [(chain-gas), 0.24])
+  (enforce-guard
```

Comment:
Here, for all other `expect-failure` tests here, and going forward, always use the new `expect-failure` that takes a string to match the error on, ie
```
(expect-failure
  (format "chain-gas {} is bigger than than max-gas {}"
    [(chain-gas), 0.24])
  "Gas Limit * Gas Price must be bigger than or equal to 0.24"
  (enforce-guard (max-gas 0.24)))
```

Reply:
Applied the changes